### ISSUE TITLE
Silent NUMBER cast to INTEGER

### ISSUE THREAD

**Issue by kshcherbatov** [tarantool/tarantool#4216]
```
tarantool> box.execute("CREATE TABLE t2(id  INT primary key, x INTEGER);")
-- on the one hand
tarantool> box.execute("INSERT INTO t2 VALUES(3, \"1.1\")")
---
- error: Can’t resolve field '1.1'
...
-- on the other
tarantool> box.execute("INSERT INTO t2 VALUES(3, 1.1)")
---
- row_count: 1
...
```

Postgress, DB2, mysql performs it in same way (also in 'ansi' mode).
@pgulutzan  please, give a feedback. Should it be fixed?

**Comment by pgulutzan** [comment]
I commented via email in thread [dev] cast, CCed to dev.

**Comment by kyukhin** [comment]
Closing then.

**Comment by Korablev77** [comment]
@kyukhin 
Why did you close this issue? I open mailing list and see answer from Peter:

> So, at last, the short answer is:
> Yes it should be fixed.

**Comment by ImeevMA** [comment]
Closed by ac5405bf2c9865f35e5e539f3a3ecbdae69c4218
